cmd/lncli/commands.go

```diff
@@ -1398,33 +1398,11 @@ func create(ctx *cli.Context) error {
 	client, cleanUp := getWalletUnlockerClient(ctx)
 	defer cleanUp()
 
-	// First, we'll prompt the user for their passphrase twice to ensure
-	// both attempts match up properly.
-	fmt.Printf("Input wallet password: ")
-	pw1, err := terminal.ReadPassword(int(syscall.Stdin))
-	if err != nil {
-		return err
-	}
-	fmt.Println()
-
-	fmt.Printf("Confirm wallet password: ")
-	pw2, err := terminal.ReadPassword(int(syscall.Stdin))
+	walletPassword, err := capturePassword("Input wallet password: ",
```

Comment:
nit: suggested wrapping
```golang
walletPassword, err := capturePassword(
        "Input wallet password: ", false, walletunlocker.ValidatePassword,
)
```
as a general rule if function calls can't fit on one line the preferred wrapping is:
```
function(
        arg1, arg2, arg3,
)
```